### PR TITLE
OCPBUGS-33898: `adm update status`: Show information about the upgrade target version

### PR DESCRIPTION
**Pull request by Davoska**
This pull request will show additional information regarding the upgrade.

A new line in the control plane section shows information regarding the upgrade target version and the previous version.

For example:
```text
= Control Plane =
Assessment:      Progressing
Target Version:  4.14.1 (from 4.14.0)
Completion:      97%
Duration:        14m4s
Operator Status: 32 Healthy, 1 Unavailable
```

In the event that the previous update was an incomplete update (a partial update) and the control plane is currently updating, a new insight is generated, as this fact may influence the upgrade.

This pull request references https://issues.redhat.com/browse/OTA-1156

### CODE REVIEW

**Comment by Davoska** [comment]
/label acknowledge-critical-fixes-only

A small but important improvement to a technology preview feature that is not tested in the CI and does not affect other components in any way. This won't destabilize the 4.16.

**Review by petr-muller — APPROVED** [review]
/retest-required

**Comment by petr-muller** [comment]
And again...
/lgtm
/approve
/meow

File: pkg/cli/admin/upgrade/status/controlplane.go [hunk 1]
```diff
@@ -197,10 +213,61 @@ func assessControlPlaneStatus(cv *v1.ClusterVersion, operators []v1.ClusterOpera
...
+		target:   "unknown",
+		previous: "unknown",
+	}
+	if len(history) > 0 {
+		versionData.target = history[0].Version
+	}
+	if len(history) > 1 {
+		versionData.previous = history[1].Version
+		versionData.isPreviousPartial = history[1].State == v1.PartialUpdate
+	}
```
Comment (wking) [inline pkg/cli/admin/upgrade/status/controlplane.go:235]:
```golang
else if len(history) == 1 {
  version data.previous = "a new install"
}
```

or something?

**Comment by petr-muller** [comment]
/retitle OCPBUGS-33898: `adm update status`: Show information about the upgrade target version

**Pull request merged**